iOSClient/Data/NCDatabase.swift

```diff
@@ -107,6 +108,14 @@ class tableAccount: Object {
     }
 }
 
+class tableActivityLatestId: Object {
+    @objc dynamic var account = ""
+    @objc dynamic var mostRecentlyLoadedActivityId: Int = 0
+    override static func primaryKey() -> String {
+        return "account"
+    }
+}
+
 class tableActivity: Object, DateCompareable {
```

Comment:
small fixation, can you move this new `tableActivityLatestId` after the class `tableActivity` (sorted by letter)